src/main/kotlin/org/objectionary/ddr/graph/InnerPropagator.kt

```diff
@@ -65,7 +65,7 @@ class InnerPropagator(
     private fun processDecorators() {
         val repetitions = 5
         // @todo #44:30min this solution is naive, optimize it (see commented out lines)
```

Comment:
@c71n93 According to our convention TODOs are written as part of JavaDoc comment for the nearest method/class. In general intra-method comments are discouraged.

Reply:
@mximp fixed this